panther_gazebo/config/led_strips.yaml

```diff
@@ -0,0 +1,21 @@
+chanel_1:
+  frequency: 15  # Setting to high frequency caused lags in the simulation
+  world_name: husarion_world
+  parent_link: panther
+  position: [0.362, 0.0, 0.201]
+  orientation: [0.0, 0.0, 0.0]
+  led_strip_width: 0.5
+  topic: lights/driver/channel_1_frame
+  light_name: front_light
+  number_of_leds: 41
```

Comment:
why this is 41?

Reply:
Good catch, I test MsgValidation function